main.py

```diff
@@ -1,52 +1,70 @@
 import uuid
 
 
-class Bank():
-    default_log='None'
-    def __init__(self,user_name,log=default_log):
-        self.user_name=user_name
-        self.log=str(log)
+class User():
+    def __init__(self,name):
+        self.name=name
         self.card=Card()
```

Comment:
Don't you need to init log with empty list here as well?(not sure - but i believe you do need.)
this is in case you will decide to move it to this class. i later found it in other place